### ISSUE TITLE
Feature request: inline conditional

### ISSUE THREAD

**Issue by norman784** [ronan-gloo/jade-php#4]
I have in my template something like 

``` jade
li(class="#{current_page=='dashboard'?'active':''}")
```

And throws `Expecting a variable name got: ='dashboard'?'active':'' (0)`, but according to this [question](http://stackoverflow.com/questions/7563647/jade-inline-conditional) seems to be possible on jade-js.

**Comment by ronan-gloo** [comment]
Inline conditions are not parsed by the engine, you're right.

**Comment by norman784** [comment]
yeah, let me know if it will be supported and / or when is supported. Or light me where need to be done the patch to make it work...

Regards
